src/main/java/TaskList.java

```diff
@@ -0,0 +1,184 @@
+        System.out.println("\n~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(count - 1).toString());
+        System.out.println("Now you have " + count
+                + (count == 1 ? " task" : " tasks")
+                + " in the list.");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
+        try {
+            Storage.saveDataToFile(tasks);
+        } catch (UnableToSaveDataException e) {
+            System.out.println("Sorry, unable to save data to the hard disk.");
+        }
+    }
+
+    /**
+     * Adds a deadline to the list.
+     *
+     * @param command Command input by the user
+     *                which contains the name of
+     *                the event and the date and time.
+     * @param tasks List of all tasks.
+     */
+
+    public static void addEvent(String command, List<Task> tasks) throws InvalidDateTimeFormatException {
+        try {
+            tasks.add(new Event(command.substring(6).split("/")[0],
+                    LocalDateTime.parse(command.substring(6).split("/")[1].substring(3),
+                            DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm"))));
+        } catch (DateTimeParseException e) {
+            throw new InvalidDateTimeFormatException();
+        }
+        int count = tasks.size();
+        System.out.println("\n~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(count - 1).toString());
+        System.out.println("Now you have " + count
+                + (count == 1 ? " task" : " tasks")
+                + " in the list.");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
+        try {
+            Storage.saveDataToFile(tasks);
+        } catch (UnableToSaveDataException e) {
+            System.out.println("Sorry, unable to save data to the hard disk.");
+        }
+    }
+
+    /**
+     * Deletes the specified task.
+     *
+     * @param command Command input by the user
+     *                which contains the index of
+     *                the task to be deleted.
+     * @param tasks List of all tasks.
+     */
+    public static void deleteTask(String command, List<Task> tasks) {
+        int deleteIndex = Integer.parseInt(command.split(" ")[1]) - 1;
+        System.out.println("\n~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println("I've removed the task:");
+        System.out.println(tasks.get(deleteIndex));
+        tasks.remove(deleteIndex);
+        int count = tasks.size();
+        System.out.println("Now you have " + count
+                + (count == 1 ? " task" : " tasks")
+                + " in the list.");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
+        try {
+            Storage.saveDataToFile(tasks);
+        } catch (UnableToSaveDataException e) {
+            Ui.printUnableToSaveDataMessage();
+        }
+    }
```

Comment:
Declaration of deleteIndex and count could be placed at the top of the deleteTask method, to easily identify where declared variables are located at.